lessons/topics/01-variables/exercises/README.md

```diff
@@ -0,0 +1,6 @@
+# My topic exercises index
+
+## Exercises
+
+* Variables: Check the differences between different declaration keywords for variables and constants
```

Comment:
I think you can resume this just by saying: Check the differences between variable declaration keywords